webapp/src/css/private-vars.less

```diff
@@ -0,0 +1,29 @@
+/**
+* WARNING:
+* Don't import this file. This is only for "variable.less" usage.
+* Don't use these variables. The variable names aren't descriptive enough.
+* Instead, use the variables from "variable.less".
+*/
```

Comment:
I couldn't find a solution to reduce Less variables' scope so this is the best we can do for now :+1:

Reply:
Cool, thanks for checking!